app/auth/user_management.py

```
@@ -0,0 +1,79 @@
+    if current_user.get("role") != "admin":
+        raise HTTPException(status_code=status.HTTP_403_FORBIDDEN, detail="Admin access required")
+    results = []
+    for user_id in request.user_ids:
+        try:
+            if request.action == "disable":
+                auth.update_user(user_id, disabled=True)
+                results.append({"user_id": user_id, "status": "disabled"})
+            elif request.action == "delete":
+                auth.delete_user(user_id)
+                results.append({"user_id": user_id, "status": "deleted"})
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)